pkg/bgpv1/manager/route_policy_reconciler.go

```diff
@@ -186,8 +192,27 @@ func (r *RoutePolicyReconciler) pathAttributesToPolicy(attrs v2alpha1api.CiliumB
 	}
 
 	switch attrs.SelectorType {
+	case v2alpha1api.CPIPKindDefinition:
+		localPools := r.populateLocalPools(params.CiliumNode)
+		for _, pool := range r.podPoolStore.List() {
+			if attrs.Selector != nil && !labelSelector.Matches(labels.Set(pool.Labels)) {
+				continue
+			}
+			// only include pool cidrs that have been allocated to the local node.
+			if cidrs, ok := localPools[pool.Name]; ok {
+				for _, cidr := range cidrs {
+					if cidr.Addr().Is4() {
+						prefix := int(pool.Spec.IPv4.MaskSize)
```

Comment:
nit: should be prefixLen, same for v6

Reply:
@harsimran-pabla thanks for the review. Commit 677c3ca updates the var name to `prefixLen`.